.github/workflows/load-test.yml

```diff
@@ -0,0 +1,108 @@
+name: Load Test
+
+on:
+  schedule:
+    - cron: "0 0 * * 1" # Run every Monday at 00:00
+  workflow_dispatch:
+    inputs:
+      test_id:
```

Comment:
❓ Is this and the other inputs needed? How / why would someone change them? I also see defaults not just here but in the code, so if they stay then the code using them can assume they have values.

Reply:
Also, what's the best practice with defaults and the required attribute? If they're required can the code constants be cleaned up later down?